ui/src/scss/_sizing.scss

```diff
@@ -127,7 +127,12 @@ $abbrev in (width: w, height: h) {
 .h-100-40 {
   height: calc(100% - 40px)!important;
 }
-
+.h-24px{
```

Comment:
px can be removed from class name since we are utilizing rem unit

Reply:
sure will commit